test/integration/store_admin_can_update_store_info_test.rb

```diff
@@ -0,0 +1,15 @@
+require 'test_helper'
+
+class StoreAdminCanUpdateStoreInfoTest < ActionDispatch::IntegrationTest
+  test ' store admin can update store info' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.